stripe/src/main/java/com/stripe/android/view/CvcEditText.kt

```diff
@@ -54,7 +54,9 @@ class CvcEditText @JvmOverloads constructor(
         filters = createFilters(CardBrand.Unknown)
 
         inputType = InputType.TYPE_NUMBER_VARIATION_PASSWORD
-        keyListener = DigitsKeyListener.getInstance(false, true)
+
+        // do not allow plus/minus sign or decimal separator
+        keyListener = DigitsKeyListener.getInstance(false, false)
```

Comment:
You should be able to achieve the same thing with `android:inputType="numberPassword"`. Did you try this?

```kotlin
inputType = InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_VARIATION_PASSWORD
```

Reply:
This will also restrict the software keyboard.

![Screenshot_1601041765](https://user-images.githubusercontent.com/68603497/94275015-72486f80-ff14-11ea-9759-fdb58f6c1123.png)